owncloudApp/src/main/java/com/owncloud/android/ui/activity/LogHistoryActivity.java

```diff
@@ -106,16 +117,99 @@ protected void onCreate(Bundle savedInstanceState) {
                 showLoadingDialog();
 
                 // Start a new thread that will load all the log data
-                LoadingLogTask task = new LoadingLogTask();
+                LoadingLogfileTask task = new LoadingLogfileTask();
                 task.execute();
             }
         }
     }
 
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        MenuInflater inflater = getMenuInflater();
+        inflater.inflate(R.menu.log_menu, menu);
+        MenuItem searchItem = menu.findItem(R.id.menu_search);
+        SearchManager searchManager = (SearchManager) getSystemService(Context.SEARCH_SERVICE);
+
+        if (searchItem != null) {
+            searchView = (SearchView) searchItem.getActionView();
+            //            searchView.setQueryHint(getString(R.string.text));
+        }
+        if (searchView != null) {
+            searchView.setSearchableInfo(searchManager.getSearchableInfo(getComponentName()));
+        }
+
+        if (searchView == null) {
+            return true;
+        }
+        SearchView.SearchAutoComplete searchAutoComplete = searchView.findViewById(R.id.search_src_text);
+
+        if (searchView != null && mCurrentFilter != null && !mCurrentFilter.equals("")) {
```

Comment:
Not sure if this check is needed, you are already checking whether `searchView is null` above and returning true if so.